### ISSUE TITLE
Webpack 3.0 support

### ISSUE THREAD

**Issue by renchap** [webpack-contrib/worker-loader#75]
The `peerDependancies` needs to be updated to allow Webpack 3.0.

After a `yarn upgrade webpack && yarn upgrade`, tests are not passing (same error for all):
```
     TypeError: Cannot read property 'map' of undefined
      at makeBundle.then (test/index.js:39:9)
      at <anonymous>
```

I am not familiar enough with Webpack's internals to provide a fix.

**Comment by mikegreiling** [comment]
this is blocking us from upgrading to webpack v3.1.0 🤞

**Comment by Reon90** [comment]
When will it go to release version?

**Comment by renchap** [comment]
Work is in progress in #81

**Comment by qbaty** [comment]
This blocks us too, But it's pr seems to be done very soon~

**Comment by HillLiu** [comment]
Please release this one.

https://github.com/webpack-contrib/worker-loader/pull/85

**Comment by ahutchings** [comment]
What is the status of this issue - can it be closed? I see that #85 is merged and released in 1.0.0.
